### DISCUSSION TITLE
HA running with Zigbee gateway

### DISCUSSION THREAD

**Discussion by kristianolofsson** [seaky/nspanel_pro_tools_apk#97] [General]
Is it possible to run the HA app and in parallel keep the ZigBee network up and running?

**Comment by jengel84** [comment]
I can't answer the original question, but here is my related question:

Is it possible to connect the NSPanel with seaky's pro tools to an existing zigbee network as a router?
Or is this planned in one of the upcoming releases?

**Reply by kristianolofsson** [reply]
I agree it would be a useful request, currently the nspanel can only serve as a router to an existing network if you disregard its usage as a hub. Meaning cutting your local connections to eg R5 scene devices.

I'd like the nspanel to act as a device in my zigbee network but also keep the sub items.

Then on top of this run HA on front end.

**Reply by CFenner** [reply]
When I set up the panel (v2.2.0) I changes to router mode (ewe app) and connected it to my existing ZHA network. No issues so far. Just note that you need to do that first, as this resets the whole panel.
